src/AppKit/NSView.cs

```diff
@@ -31,20 +33,25 @@ sealed class SortData {
 #endif
 		static nint view_compare (IntPtr view1, IntPtr view2, IntPtr context)
 		{
-			var data = (SortData) GCHandle.FromIntPtr (context).Target;
+			var data = GCHandle.FromIntPtr (context).Target as SortData;
```

Comment:
Why was this change necessary? `data` should always be `SortData`, so the old code is safe.